cpp/test/neighbors/ann_ivf_flat.cuh

```diff
@@ -156,7 +156,7 @@ class AnnIVFFlatTest : public ::testing::TestWithParam<AnnIvfFlatInputs<IdxT>> {
         auto database_view = raft::make_device_matrix_view<const DataT, IdxT>(
```

Comment:
We need to include `raft/neighbors/specializations.cuh` in this file for `#ifdef RAFT_DISTANCE_COMPILED`

Reply:
This one was already including that, fortunately.